bin/p2-exec/main.go

```diff
@@ -110,6 +111,12 @@ func main() {
 		}
 	}
 
+	if *requireFile != "" {
+		if _, err := os.Stat(*requireFile); os.IsNotExist(err) {
```

Comment:
Please log the other error cases. I don't those sorts of things should crash `p2-exec`, but I could be convinced.